src/core/or/policies.c

```diff
@@ -2817,12 +2818,13 @@ parse_short_policy(const char *summary)
       sizeof(short_policy_entry_t)*(n_entries);
     result = tor_malloc_zero(size);
 
-    tor_assert( (char*)&result->entries[n_entries-1] < ((char*)result)+size);
+    tor_assert((char *) &result->entries[n_entries - 1] <
+               ((char *) result) + size);
```

Comment:
Same as above. This shouldn't be split on multiple lines, and I don't think we should have a space between the type-cast and the variables.